pyproject.toml

```diff
@@ -76,7 +75,7 @@ dev = [
     "packaging >= 24.1, < 26", # Used in test_version.py
     "pip >= 24.3.1, < 26", # Used in test_version.py
     "pre-commit >= 3.5.0, < 5",
-    "pytest-asyncio >= 0.23.8, < 0.27",
+    "pytest-asyncio >= 0.25.0, < 0.27",
```

Comment:
These version bumps aren't needed to drop support for 3.8.
I can remove them if it is desired to keep the change minimal.

Reply:
if you want I'd just do it the latest, like, `~= 0.26.0`

Reply:
basically we lock these so people don't have to figure out when something breaks when it's simply re-locked.  But if someone is actively looking at it feel free to update to latest.